### PR TITLE
Release `2024.6.0`

### PR DESCRIPTION
**Pull request by PeterNerlich**
# Release notes

- [ [#2714](https://github.com/digitalfabrik/integreat-cms/issues/2714) ] Accept longer links while central link replacement
- [ [#2756](https://github.com/digitalfabrik/integreat-cms/issues/2756) ] Use rounded hix value
- [ [#2769](https://github.com/digitalfabrik/integreat-cms/issues/2769) ] Re-add barrier-free and organization box to POI
- [ [#2783](https://github.com/digitalfabrik/integreat-cms/issues/2783) ] Mark locations as closed by default during CSV import

__________________________________________________

[Pull Request Review Guidelines](https://digitalfabrik.github.io/integreat-cms/pull-request-review-guide.html)

### CODE REVIEW

**Comment by JoeyStk** [comment]
We should probably wait for #2784

**Comment by svenseeberg** [comment]
We need a release ASAP after https://github.com/digitalfabrik/integreat-cms/pull/2787 has been merged.

File: integreat_cms/gvz_api/utils.py [hunk 1]
```diff
@@ -182,21 +182,22 @@ def __init__(
...
         if (
-            not region_name
+            region_name
             and not region_ags
             and (best_match := api.best_match(region_name, region_type))
         ):
             self.ags = best_match["ags"]
 
-        self.name: str = ""
+        self.name: str = "" if not region_name else region_name
```
Comment (cclauss) [inline integreat_cms/gvz_api/utils.py:191]:
```suggestion
        self.name: str = region_name or ""
```

File: integreat_cms/gvz_api/utils.py [hunk 2]
```diff
@@ -182,21 +182,22 @@ def __init__(
...
         self.longitude = None
         self.latitude = None
         self.child_coordinates = {}
 
         if self.ags and (details := api.get_details(self.ags)):
             self.id = details["id"]
             self.ags = details["ags"]
-            self.name = details["name"]
+            if not self.name:
+                self.name = details["name"]
```
Comment (cclauss) [inline integreat_cms/gvz_api/utils.py:200]:
```suggestion
            self.name = self.name or details["name"]
```

**Comment by PeterNerlich** [comment]
@cclauss Thank you for your suggestions! Sorry that it took so long to react to them.

> These monthly releases might be a good time to run: % `pre-commit autoupdate` % `pre-commit run --all-files`

When I tried this originally, something broke on my machine and I did not try fixing it right away. But we're having a team event right now and will probably have this done by today or tomorrow.

For your code suggestions, could you open a dedicated PR? We would like to restrict commits on `develop` to merge PRs from feature or bugfix branches, and commits on `main` to merges from `develop` only.

File: integreat_cms/gvz_api/utils.py (hunk 1 above)
Comment (timobrembeck) [reply integreat_cms/gvz_api/utils.py:191]: Thanks @cclauss! I moved the suggestions to #2823.

**Comment by timobrembeck** [comment]
I suggest to wait for #2819 to be merged and then release a new version! :rocket:

**Review by timobrembeck — APPROVED** [review]
After sitting over a month, I think this is finally good to go :joy:

**Pull request merged**